pkg/providers/vsphere/session/session_vm_update_test.go

```diff
@@ -1553,8 +1553,8 @@ var _ = Describe("UpdateVirtualMachine", func() {
 									Name: vmiName,
 									Kind: vmiKind,
 								},
-								AllowGuestControl: true,
-								Connected:         true,
+								AllowGuestControl: &[]bool{true}[0],
```

Comment:
Looks like no `ptrOf` existing in this package, but you can also use `vimtypes.NewBool(true)`

Reply:
Looks like we have a similar function provided in `pkg/util/ptr`, will replace it with that.

Reply:
Done.